HousingSearchApi.Tests/V1/Boundary/Requests/Validation/GetProcessListRequestValidatorTests.cs

```diff
@@ -39,14 +39,14 @@ public void ShouldNotErrorWithOnlySearchText()
         }
 
         [Fact]
-        public void ShouldNotErrorOnlyTargetId()
+        public void ShouldErrorOnlyTargetId()
```

Comment:
```suggestion
        public void ShouldErrorIfTargetTypeWithoutTargetId()
```
So that it makes sense why the validation error is for TargetType